Tests/UnitTests/Identity/IdentityManagerTests.swift

```diff
@@ -25,7 +26,7 @@ class IdentityManagerTests: TestCase {
 
     override func setUpWithError() throws {
         try super.setUpWithError()
-
+        self.mockIdentityAPI = mockBackend.identity as? MockIdentityAPI
```

Comment:
To avoid a potential crash in the test later, I'd change this to:
```suggestion
        self.mockIdentityAPI = try XCTUnwrap(mockBackend.identity as? MockIdentityAPI)
```
So this failure is caught early.

Reply:
💯